code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -199,6 +199,9 @@
 			FindTarget()
 
 /mob/living/simple_animal/hostile/proc/AttackingTarget()
+	if(target == src)
+		to_chat(src,"Why attack yourself?")

```

Comment:
Without this message, a player controlling a simple_animal, and clicking a window in its own tile, would receive no feedback whatsoever. They'd probably click it several times, not realizing what's wrong. With this message, it prompts them to look at the status text in the bottom left which tells them what they're actually attacking - and makes them realize that clicking anything in their tile attacks themselves. I could have phrased it as "Attacking yourself does not make sense". But I want them to look at it, and understand what's going on. Phrasing it as a question makes them, I think, more likely to look at it and take the time to understand how it works.